spk/homeassistant/src/service-setup.sh

```diff
@@ -2,7 +2,7 @@
 # - installer
 # - start-stop-status
 
-PYTHON_DIR="/usr/local/python3"
+PYTHON_DIR="/var/packages/python38/target"
```

Comment:
Just out of curiosity, what is the difference between **/usr/local/python38** and **/var/packages/python38/target**, they both seem to be a symbolic link which reference to **/volume1/@appstore/python38**.

Reply:
`/usr/local/{package}` is created by the generic installer, but it will not be supported by DSM 7+, therefore I start to use `/var/packages/{package}/target` instead. 
Such links are independent on matter whether the packages are installed on `/volume1`, `/volume2`, etc.

Reply:
@hgy59 Thanks for the explanation, much appreciated!